src/modules/map/components/openlayers/OpenLayersMap.vue

```diff
@@ -240,22 +240,34 @@ export default {
     // let's watch changes for center and zoom, and animate what has changed with a small easing
     watch: {
         center() {
-            this.view.animate({
-                center: this.center,
-                duration: 250,
-            })
+            if (!IS_TESTING_WITH_CYPRESS) {
+                this.view.animate({
+                    center: this.center,
+                    duration: 250,
+                })
+            } else {
+                this.view.setCenter(this.center)
+            }
```

Comment:
With this 3 animate issue, would it be good to define the duration as a variable, and if the flag regarding Cypress testing is true, we set the duration to 0?
That would reduce the number of specific case we have for Cypress testing.